types/node/process.d.ts

```diff
@@ -450,12 +451,13 @@ declare module 'process' {
                 listeners(event: "removeListener"): RemoveListenerListener[];
                 listeners(event: "multipleResolves"): MultipleResolveListener[];
             }
-
-            interface Global {
-                process: Process;
-            }
         }
     }
 
     export = process;
 }
+
+declare module 'node:process' {
+    import * as process from 'process';
+    export = process;
```

Comment:
Since `process` uses `export =`, I recommend using `import =` here for consistency:

```suggestion
    import process = require('process');
    export = process;
```